vars/matrix.yml

```diff
@@ -0,0 +1,10 @@
+---
+
+matrix_username: matrix
+git_repo: https://github.com/matrix-org/synapse/tarball/master
+hostname: zeus.ugent.be
+enable_registration: true
+enable_registration_captcha: false
+recaptcha_private_key: YOURPRIVATEKEYHERE
```

Comment:
Is this changed on the server? I guess not as this would be overridden on every Ansible run.
Is this überhaupt used?